content/src/main/content/jcr_root/_oak_index/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Im thinking this should be a lucene index.. they tend to be more robust than property indexes. 
Does this HAVE to be part of the commit hook?

Also - dont recall the oak version it was introduced, but lucene can now be marked as "nrt" (or near real time) .. which triggers from, though doesnt block, the commit hook.